packages/react-native-sdk/CONTRIBUTING.md

```diff
@@ -0,0 +1,17 @@
+We're really glad you're reading this, because we need volunteer developers to help this project come to fruition. 👏
+
+## Instructions
+
+These steps will guide you through contributing to this project:
+
+- Fork the repo
+- Clone it and install dependencies
+
+		git clone https://github.com/YOUR-USERNAME/typescript-library-starter
+		npm install
+
+Keep in mind that after running `npm install` the git repo is reset. So a good way to cope with this is to have a copy of the folder to push the changes, and the other to try them.
+
+Make and commit your changes. Make sure the commands npm run build and npm run test:prod are working.
+
+Finally send a [GitHub Pull Request](https://github.com/alexjoverm/typescript-library-starter/compare?expand=1) with a clear list of what you've done (read more [about pull requests](https://help.github.com/articles/about-pull-requests/)). Make sure all of your commits are atomic (one feature per commit).
```

Comment:
Is the link for **GitHub Pull Request** correct above?

Reply:
No not at all 😆. They are fixed now with https://github.com/Kinvey/js-sdk/pull/420/commits/30255e8c370c4e63a47c42347d23851b7c31dae1.